File: src/main/java/com/ivaneye/intellijweb2/mapper/OrdersMapper.java
package com.ivaneye.intellijweb2.mapper;

import com.ivaneye.intellijweb2.model.Orders;

/**
 * <p>
 *  Mapper 接口
 * </p>
 *
 * @author wangyifan
 * @since 2018-02-12
 */
public interface OrdersMapper {
    Orders selectByPrimaryKey(Long recId);
}

File: src/main/kotlin/com/ivaneye/intellijweb2/mapper/OrdersMapper.kt
package com.ivaneye.intellijweb2.mapper

import com.ivaneye.intellijweb2.model.Orders
import org.apache.ibatis.annotations.Param

/**
 * Created by wangyifan on 18-2-19.
 */
interface OrdersMapper {
    fun selectByPrimaryKey(@Param("recId") recId: Long): Orders
}